src/screens/DiscoveryScreenV2/elements/DiscoveryItemList.js

```diff
@@ -242,6 +242,7 @@ const styles = StyleSheet.create({
   },
   buttonFollowing: {
     width: normalize(65),
+    paddingLeft: 1,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding a `paddingLeft` of 1 to the `buttonFollowing` style may cause the text inside the button to be off-center. If you want to maintain the centered alignment of the content within the button, consider adding equal padding to both sides.
```diff
-    paddingLeft: 1,
+    paddingLeft: 1,
+    paddingRight: 1,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->